### DISCUSSION TITLE
The example provider-reference files under price-transparency-guide/examples are not current

### DISCUSSION THREAD

**Discussion by olderthandirt5** [CMSgov/price-transparency-guide#435] [Q&A] [answered]
The example provider-reference files are for the v0.10.1 schema, which does not match the validator. This has caused concern from our partners who have not set up the JSON Validator tool and are relying on the example files as 'gospel'. The partner says our file is incorrect even though it has been validated successfully.
The only examples that appear to be current are the allowed-amounts files.

**Comment by shaselton-usds** [comment]
Can you provide an example of your file that is in question?

Also, the provider-reference example file was last updated when v0.10.1 schema was tagged, it doesn't mean that it is invalid for v1.0. The example files should be correct but should not be the 'gospel'. The schema files themselves are what development needs to be based on, anything other than that is assuming risk.

I ran the validator against the example file for v1.0.0 and it appears to be valid.

**Comment by olderthandirt5** [comment]
Our issue was that our client was treating the example files as 'gospel'. I had not tested the example file, so I am sorry if I wasted your time.
We eventually got the client to understand the issue, and I appreciate having your explanation handy in case this comes up again.
The validator tool is indispensable, and we have had nothing but good experience with it.
Thanks for your response.

**Comment by shaselton-usds** [comment, marked as answer]
Great -- no sweat.

Glad to hear the validator tool is helping out.

**Comment by wsmjr** [comment]
@shaselton-usds, hello and observation/comment, for the provider reference schema and example, is it worthwhile to update to show the implementation of the provider group id attribute?

1. Update https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/provider-reference/provider-reference.json to show
2. Update https://github.com/CMSgov/price-transparency-guide/blob/master/examples/provider-reference/provider-reference.json to show

Maybe a commit/PR has not been pushed for this but would help get things more in sync for those seeing things here for the first time and implementing this provider reference extract for eventual consumption :) Cheers!

**Reply by wsmjr** [reply]
@shaselton-usds also just found via https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#provider-reference-object that all of the links under the Provider Reference Object > location attribute are broken (404) as well.

**Reply by shaselton-usds** [reply]
@wsmjr Thanks for the heads up. My mistake for having the links point to a branch vs a commit and thus would become stale once the branch was deleted. 

Take a peek at: https://github.com/CMSgov/price-transparency-guide/pull/445 That should clear up the links that you've pointed out.

**Reply by wsmjr** [reply]
@shaselton-usds you rock good sir, much appreciated!
